core-bundle/contao/languages/en/tl_preview_link.xlf

```diff
@@ -75,7 +75,7 @@
         <source>Please save this record to generate the preview link.</source>
       </trans-unit>
       <trans-unit id="tl_preview_link.hintEdit">
-        <source>Preview link: &lt;a href="%s" data-to-clipboard="%s"&gt;%s&lt;/a&gt;</source>
+        <source>Preview link</source>
```

Comment:
This to me is the main advantage of using Stimulus :)

Reply:
absolutely not related to Stimulus 🤣